core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -313,6 +313,10 @@ public String toString() {
         return sb.toString();
     }
 
+    public String displayVersion() {
```

Comment:
I foresee being confused about whether we should use this or `toString`. Maybe we should just use `toString` in this case?